grr/gui/api_client/client.py

```diff
@@ -56,6 +56,27 @@ def ListFlows(self, offset=0, count=0):
     return utils.MapItemsIterator(
         lambda data: flow.Flow(data=data, context=self._context),
         items)
+ 
+  def HasValidApproval(self):
+    """Determine if valid approval exists for this client."""
+  
+    args = api_pb2.ApiListClientApprovalsArgs(
+        offset=0, count=0, client_id=self.client_id, state=api_pb2.VALID)
```

Comment:
There's probably no need to initialize offset and count, we can just leave them out.